Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ however, it has to be formatted properly to pass verification tests.
 
 ### Fixed
 - Partially fixed case ISX-1357 (Investigate performance regressing over time).  A sample showed that leaving an InputActionMap enabled could lead to an internal list of listeners growing.  This leads to slow-down, so we now warn if we think this is happening.
+- Fixed case ISXB-580 (UI Submit / Cancel not working with Switch Pro controller) by adding "Submit" & "Cancel" usages to the Switch Pro controller input controls.
```

Comment:
Suggest providing ISXB-580 as a hyperlink to the issue tracker since this is what we have done previously. This provides a convenient way to check the corresponding issue that the fix-claim references. In this case e.g. https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-580

Reply:
Also noted that CHANGELOG.md has a conflict that needs to be resolved.

Reply:
Resolved conflicts and added link to issue tracker in changelog in 827232f.